### PR TITLE
Service Now Integration

### PR DESCRIPTION
**Pull request by skoyev**
Service Now Integration with CX Flow

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ logging:
 cx-flow:
   # Agreed upon shared API token
   token: xxxx
-  bug-tracker: JIRA
+  bug-tracker: ServiceNow
```
Comment (kmcdon83) [inline src/main/resources/application.yml:42]: Can you remove this and leave as JIRA

File: src/main/resources/application.yml [hunk 2]
```diff
@@ -118,6 +119,15 @@ rally:
...
   api-url: https://rally1.rallydev.com/slm/webservice/v2.0
 
+servicenow:
+  token: 123
+  servicenow-project-id: xxxx
+  servicenow-workspace-id: xxxx
+  url: https://servicenow.com
+  apiUrl: https://dev90318.service-now.com/api/now/table
+  username: admin
+  password: ServiceNow123!
```
Comment (kmcdon83) [inline src/main/resources/application.yml:129]: Replace with xxxx

File: src/main/resources/application.yml (hunk 1 above)
Comment (skoyev) [reply src/main/resources/application.yml]: Done

File: src/main/resources/application.yml (hunk 2 above)
Comment (skoyev) [reply src/main/resources/application.yml]: Done

**Comment by AvivCX** [comment]
@skoyev  please see Sonar checks

**Pull request merged**